galeracluster/source/documentation/architecture.rst

```diff
@@ -73,7 +73,7 @@ The internal architecture of Galera Cluster revolves around four components:
 
 - **Galera Replication Plugin:** This plugin enables write-set replication service functionality.
 
-- **Group Communication Plugins:** There several group communication systems available to Galera Cluster (e.g., *gcomm* and `Spread <https://www.spread.org/>`_).
+- **Group Communication Plugins:** The initial idea was to have several group communication systems available to Galera Cluster (e.g., *gcomm* and `Spread <https://www.spread.org/>`_), however for practicality the only group communication system one uses with Galera Cluster is *gcomm*.

```

Comment:
Is it necessary event mention "Group Communication Plugins" here? Even though some of the plugin infra is still there, there is only one group communication implementation, which is one component of "Galera Replication Plugin".

I would simply remove group communication plugins from here and from the picture, it does not add any relevent information.

Reply:
The reason I keep it is because there is *hope* that someone may consider having it if they ever wanted to extend Galera Cluster. Though as you say, it hasn't happened in ~15 years, so maybe it truly doesn't matter